src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2373,6 +2470,14 @@ public static function get_tests() {
 			);
 		}
 
+		// Only check for a persistent object cache in production environments to not unnecessarily promote complicated setups.
+		if ( 'production' === wp_get_environment_type() ) {
+			$tests['direct']['persistent_object_cache'] = array(
+				'label' => __( 'Persistent object cache' ),
+				'test'  => 'health_persistent_object_cache',
```

Comment:
The `health_` prefix here and elsewhere does not make sense, let's remove it throughout this name and the relevant methods.
```suggestion
				'test'  => 'persistent_object_cache',
```